### DISCUSSION TITLE
mechanism to pre-fetch full articles in bulk

### DISCUSSION THREAD

**Discussion by khimaros** [spacecowboy/Feeder#340] [Ideas]
if i understand correctly, the current behavior, even for feeds which have the "fetch full article" enabled, syncing only fetches the RSS content from the feed. this makes some feeds unreadable offline. instead it is necessary to open or "save" each article that i want to be able to read offline.

it would be great if there were a way to pre-cache the full articles to allow for offline reading. it would also be fine if this were a separate button/action rather than always being the default sync or even a separate action per-feed.

**Comment by spacecowboy** [comment]
Enabling "full text by default " will fetch them during background sync so this is already implemented
